spec/git_helper_spec.rb

```diff
@@ -132,6 +132,23 @@
       end
       # rubocop:enable RSpec/ExampleLength
     end
+
+    it 'returns true when the path is outside the repository folder' do
+      # This path is in the parent directory, which is not a Git repo
+      path = File.join(@path, '..', 'dummy.txt')
```

Comment:
🤔 maybe the reason the tests fails has to do with how `@path` is generated in macOS vs the CI environment where they run?